src/components/note/note.pw.tsx

```diff
@@ -0,0 +1,252 @@
+  });
+});
+
+test.describe("check action events for Note component", () => {
+  test("should call onLinkAdded callback when a valid url is detected by Note component", async ({
+    mount,
+    page,
+  }) => {
+    let hasOnLinkAddedBeenCalledCount = 0;
+
+    await mount(
+      <NoteComponent
+        text="https://carbon.s"
+        onLinkAdded={() => {
+          hasOnLinkAddedBeenCalledCount += 1;
+        }}
+      />
+    );
+
+    await linkPreview(page).waitFor();
+    expect(hasOnLinkAddedBeenCalledCount).toBe(1);
+  });
+});
+
+test.describe("Accessibility tests for Note component", () => {
+  testData.forEach((text) => {
+    test(`should render Note with noteContent prop as ${text} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent text={text} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [{ width: 30 }, { width: 75 }].forEach(({ width }) => {
+    test(`should render Note with width prop is set to ${width} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent width={width} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  test("should render Note with inlineControl prop for accessibility tests", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<NoteComponentWithInlineControl />);
+
+    await checkAccessibility(page);
+  });
+
+  testData.forEach((title) => {
+    test(`should render Note with title prop set to ${title} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent title={title} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((name) => {
+    test(`should render Note with name prop set to ${name} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent name={name} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  test("should render Note with createdDate prop for accessibility tests", async ({
+    mount,
+    page,
+  }) => {
+    const createdDate = "25 June 2022, 11:57 AM" as string;
+
+    await mount(<NoteComponent createdDate={createdDate} />);
+
+    await checkAccessibility(page);
+  });
+
+  testData.forEach((value) => {
+    test(`should render Note with status prop set to ${value} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NoteComponent
+          status={{
+            text: value,
+            timeStamp: `${CHARACTERS.STANDARD}`,
+          }}
+        />
+      );
+
+      await checkAccessibility(page);
+    });
+  });
+
+  test("should render Note with previews prop for accessibility tests", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<NoteComponentWithPreviews />);
+
+    await checkAccessibility(page);
+  });
+});
+
+test("should render with expected border radius styling", async ({
```

Comment:
non-blocking: think it could look better if this test could under something like test.describe("check expected border radius styling")

Reply:
I did wonder about this, as did Divya. It was out on its own in the original Cypress file, presumably because there's not much benefit in having another describe for one test. If there was another styling type test then I'd have put it in a describe. I'll ask in our QA channel for others' opinions on this. We should have a standard.

Reply:
Following discussion in the Slack channel, I've decided to leave that test outside of a `describe` as there are no others like it to group it with. I would definitely have put it into a `describe` if there was another radius test case or some other styling test case to group it with though.

Reply:
After thinking about this, and the fact that the only action event test case is within its own `describe`, I've moved this to within its own `describe` reading to styling. I think we should adopt this pattern for other components. So thanks for mentioning it, and also to @divyajindel for querying the same :)